### PR TITLE
Add result for 2-way PCA.

### PR DESCRIPTION
**Pull request by deflaux**
Also added a note about the UDF version of Hardy-Weinberg Equilibrium.

### CODE REVIEW

File: R/PlatinumGenomes-QC/Variant-Level-QC.Rmd [hunk 1]
```diff
@@ -182,6 +182,8 @@ if(is.null(result)) {
 }
 ```
 
+See also [a version of this query](./sql/hardy-weinberg-udf.sql) that uses the new BigQuery feature of user-defined javascript functions which is currently in limited preview.  For more info, see https://www.youtube.com/watch?v=GrD7ymUPt3M#t=1377.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Variant-Level-QC.Rmd]: Cloud deprecated the terms "Preview" and "Limited Preview":
 http://googlecloudplatform.blogspot.com/2014/10/new-release-phases-for-google-cloud.html

"Preview" is now "Beta" and "Limited Preview" is "Alpha".

File: R/PlatinumGenomes-QC/Variant-Level-QC.md [hunk 2]
```diff
@@ -463,6 +463,8 @@ Displaying the first few rows of the dataframe of results:
   <tr> <td> chr1 </td> <td align="right"> 110191193 </td> <td> T </td> <td> C </td> <td align="right">   9 </td> <td align="right">   0 </td> <td align="right">   8 </td> <td align="right"> 4.76 </td> <td align="right"> 8.47 </td> <td align="right"> 3.76 </td> <td align="right"> 17.03 </td> <td> TRUE </td> </tr>
    </table>
 
+See also [a version of this query](./sql/hardy-weinberg-udf.sql) that uses the new BigQuery feature of user-defined javascript functions which is currently in limited preview.  For more info, see https://www.youtube.com/watch?v=GrD7ymUPt3M#t=1377.
```
Comment (mbookman) [inline R/PlatinumGenomes-QC/Variant-Level-QC.md]: Use "Alpha release"

**Comment by mbookman** [comment]
Two small things regarding the use of "limited preview".

**Comment by pgrosu** [comment]
Why not just use `EVERY` instead of `SOME` and `NOT SOME` to simplify the query for lines [17-18 of gender-check.sql](https://github.com/deflaux/codelabs/blob/e0a0a05090e97bbb3c882a718dd682b25b5130a5/R/PlatinumGenomes-QC/sql/gender-check.sql#L17-18):  

```
EVERY(call.genotype = 0) WITHIN call AS hom_RR,
EVERY(call.genotype > 0) WITHIN call AS hom_AA,
```

With `SOME` being present, I'm surprised that the keywords `ONLY`, `BOTH`, `ANY` and `ALL` don't exist in BigQuery.  Are there other ways besides JavaScript for creating custom functions in BigQuery?

Thanks,
~p

**Pull request merged**